02-Context-or-Narrative/learner-paths-to-ps.graphml

```diff
@@ -0,0 +1,415 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Which editor do you use for this file type?

Reply:
https://www.yworks.com/products/yed/download#download

Reply:
Oh cool. We'll have to remember to add a link to this somewhere in the project, if you feel it's worth recommending others to use it. I'll make an issue for it.

Reply:
I made this issue: https://github.com/chexxor/purescript-documentation-discussion/issues/57
So I'll resolve this thread.

Reply:
Ok. Sounds good!